mdata/store_cassandra.go

```diff
@@ -435,12 +449,20 @@ func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]ch
 	results := make(chan outcome, numQueries)
 	for i := range crrs {
 		crrs[i].out = results
-		c.readQueue <- crrs[i]
+		select {
+		case c.readQueue <- crrs[i]:
+		default:
+			cassReadQueueFull.Inc()
+			return nil, errReadQueueFull
+		}
```

Comment:
@dieter your math on queue size is way off.  You are dreaming to think you can query cassandra at 96million requests per second.

The entire purpose of this PR is to prevent MT instances from trying to send too many queries to cassandra.  So base the queue sizes on the read performance of cassandra.

We really dont MT instances querying more than about 5k/second per MT node.  So a single query split across 4 shards would be able to query at 20k/second.

So the queue should not be more then about 200k (10seconds at 4x the average rate)
Increasing anymore then that will just lead to overloading cassandra.